test/e2e/data/e2e_conf.yaml

```diff
@@ -116,8 +116,8 @@ providers:
   - name: aws
     type: InfrastructureProvider
     versions:
-      - name: v1.5.2 # latest published release in the v1beta1 series; this is used for v1beta1 --> v1beta2 clusterctl upgrades test only.
-        value: "https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v1.5.2/infrastructure-components.yaml"
+      - name: v1.5.3 # latest published release in the v1beta1 series; this is used for v1beta1 --> v1beta2 clusterctl upgrades test only.
+        value: "https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v1.5.3/infrastructure-components.yaml"
```

Comment:
This shouldn't be changed – it's the CAPA version, not the CAPI version (see test failure)

Reply:
nice catch